api/api/utils/aiohttp.py

```diff
@@ -0,0 +1,15 @@
+import aiohttp
+
+
+# aiohttp recommends reusing the same session for the whole application
+# https://docs.aiohttp.org/en/stable/client_quickstart.html#make-a-request
+_SESSION: aiohttp.ClientSession = None
+
+
+def get_aiohttp_session():
+    global _SESSION
+
+    if _SESSION is None or _SESSION.loop.is_closed():
+        _SESSION = aiohttp.ClientSession()
```

Comment:
Checking the event loop makes unit testing easier and is extra security for cases where the event loop closed unexpectedly. If the loop associated with the session is closed, we _need_ a new session or it'll try to use the closed event loop.